packages/account/src/providers/provider.ts

```diff
@@ -441,9 +441,9 @@ export default class Provider {
       checkFuelCoreVersionCompatibility(nodeInfo.nodeVersion);
 
     if (!isMajorSupported || !isMinorSupported) {
-      throw new FuelError(
-        FuelError.CODES.UNSUPPORTED_FUEL_CLIENT_VERSION,
-        `Fuel client version: ${nodeInfo.nodeVersion}, Supported version: ${supportedVersion}`
+      // eslint-disable-next-line no-console
+      console.warn(
+        `Unsupported Fuel client version: ${nodeInfo.nodeVersion}, Supported version: ${supportedVersion}`
```

Comment:
Maybe we could improve the warning by replacing it with something like:

> The Fuel Node that you are trying to connect to is using a `fuel-core` version that is incompatible with the version of the TS SDK that you are using.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2209/commits/49c48fa9742daf1ef3e1ebd2df4852b3761834d1